kolibri_dynamic_collections_plugin/assets/src/modules/coreCollections/utils.js

```diff
@@ -1,19 +1,47 @@
 import { ContentNodeKinds } from 'kolibri.coreVue.vuex.constants';
 import { getContentNodeThumbnail } from 'kolibri.utils.contentNode';
 import tail from 'lodash/tail';
+import { AllContentNodeResource } from '../../api-resources';
+
+export function loadContentNodeDetails(node) {
+  // Inefficiently fetch additional details about a content node, which
+  // includes asking the server for a list of all of its children.
+  // FIXME: Replace this with a custom API that returns the data we need using
+  //        a single request.
```

Comment:
Is there an issue filed for this FIXME? If so, it could be linked here to make follow-up a bit easier

Reply:
Ah, yes, it definitely should! Thanks for noticing :)